tvb_library/tvb/basic/neotraits/info.py

```diff
@@ -33,6 +33,7 @@
 import numpy
 import typing
 
+from docrepr import sphinxify
```

Comment:
I look at the many dependencies of Sphinx https://github.com/sphinx-doc/sphinx/blob/master/pyproject.toml
Also at its size (6.7MB) on Pypi https://pypi.org/project/Sphinx/#files

Although license is BSD (3 clauses) for Sphinx, I would give a big vote to stay with `docutils` instead of `docrepr` as new dependency to add for tvb-library, for being much lighter.

Also, as we are packaging TVB for EBRAINS in Spack,  `docrepr` would not exist https://packages.spack.io/ and we would need to pack it, as well as maybe some of its dependencies, while `docutils` already exists.

Reply:
I dropped the docrepr commit